views/lib/forms/form-create-videoconference.hbs

```diff
@@ -6,17 +6,17 @@
 	<div class="create-videoconference">
 		<div class="checkbox create-videoconference-option">
 			<p id="startMuted" class="create-videoconference-option-text">{{$t "lib.form_create_videoConference.text.muteParticipants" }}</p>
-            <input aria-labelledby="startMuted" name="startMuted" type="checkbox" data-toggle="toggle" data-on="{{$t "global.button.yes" }}" data-off="{{$t "global.button.no" }}">
+            <input aria-labelledby="startMuted" name="startMuted" type="checkbox" data-toggle="toggle" data-on="{{$t "global.button.yes" }}" data-off="{{$t "global.button.no" }}" data-testid="toggle_mute_participants" checked>
```

Comment:
Is it correct to add "checked" here?

Reply:
you're right, will fix that as discussed